bundles/config/org.eclipse.smarthome.config.xml/config-description-1.0.0.xsd

```diff
@@ -1,57 +1,80 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" 
```

Comment:
Please check the formatter here. Looks like you changed a lot of spaces to tabs.